### PR TITLE
Fix ATIS request and to change message into small text so its readable.

### PR DESCRIPTION
**Pull request by parlock**

### CODE REVIEW

File: salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/FMCMainDisplay.js [hunk 1]
```diff
@@ -287,17 +298,53 @@ class FMCMainDisplay extends BaseAirliners {
...
+                } else {
                     this.setLine(template[tIndex][0], i, -1);
                 }
             }
         }
         if (template[13]) {
             this.setInOut(template[13][0]);
         }
-        SimVar.SetSimVarValue("L:AIRLINER_MCDU_CURRENT_FPLN_WAYPOINT", "number", this.currentFlightPlanWaypointIndex);
+        SimVar.SetSimVarValue("L:AIRLINER_MCDU_CURRENT_FPLN_WAYPOINT", "number", this.currentFlightPlanWaypointIndex).then();
```
Comment (ninjomcs) [inline salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/FMCMainDisplay.js:309]: why the .then()?

Comment (parlock) [reply salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/FMCMainDisplay.js:309]: Typo.  I am new to this pull request stuff.  how do you recommend I send you an update to remove that from the pull request?

Comment (ninjomcs) [reply salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/FMCMainDisplay.js:309]: switch to the "messages" branch on your fork, commit and push and the PR should update

Comment (ninjomcs) [reply salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/FMCMainDisplay.js]: did it for you

**Review by ninjomcs — APPROVED** [review]

**Pull request merged**